lms/djangoapps/courseware/date_summary.py

```diff
@@ -149,7 +152,7 @@ class TodaysDate(DateSummary):
 
     @property
     def date_format(self):
-        return u'%b %d, %Y (%H:%M {tz_abbr})'.format(tz_abbr=get_time_zone_abbr(self.time_zone))
+        return u'%b %d, %Y (%H:%M {tz_abbr})'.format(tz_abbr=self.time_zone.zone)
```

Comment:
This is code that will be moved into the templates, in a subsequent ticket, right?